src/ripple/ledger/PaymentSandbox.h

```diff
@@ -188,6 +188,13 @@ class PaymentSandbox final
     apply (PaymentSandbox& to);
     /** @} */
 
+    // Return a map of balance changes on trust lines and amount of XRP destoryed.
+    // The low account is be the first account in the key.
+    // If the two accounts are equal, the map contains the total changes in currency
+    // regargless of issuer. This is useful to get the total change in XRP balances.
+    std::pair<std::map<std::tuple<AccountID, AccountID, Currency>, STAmount>, XRPAmount>
```

Comment:
@vinniefalco: Yeah, these signatures will be ugly.  But if we can sensibly disentangle a bit of it that will help.  If there's a performance benefit to the `pair` return value then it makes sense to leave this as it is.